src/main/resources/DatabaseSchema.sql

```diff
@@ -0,0 +1,108 @@
+-- MySQL dump 10.13  Distrib 8.0.11, for macos10.13 (x86_64)
+--
+-- Host: localhost    Database: assignment4
+-- ------------------------------------------------------
+-- Server version	8.0.25
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
```

Comment:
use snake case

Reply:
add metadata `created_at` and `updated_at`

Reply:
Created the new fields of creation time and updation time. And used snake case.